plugins/candela/package.json

```diff
@@ -2,7 +2,7 @@
     "name": "candela-plugin",
     "version": "0.1.0",
     "dependencies": {
-        "candela": "^0.12.1",
+        "candela": "^0.16.0",
```

Comment:
I don't think plugins should necessarily be bound by this rule, but [core Girder uses the tilde (`~`) range specifier for packages that are bundled into the web client](http://girder.readthedocs.io/en/latest/development.html#modifying-core-web-client-libraries).